lms/static/certificates/sass/_components.scss

```diff
@@ -161,7 +161,7 @@
 
 // main accomplishment
 .accomplishment-main {
-    background: palette(grayscale-cool, x-trans);
+    background: transparentize(palette(grayscale, back), 0.75);
```

Comment:
The previous version used ``x-trans`` which makes it 75% transparent, so I was mimicking that.